darts/models/torch_forecasting_model.py

```diff
@@ -1063,6 +1063,9 @@ def _basic_compare_sample(train_sample: Tuple, predict_sample: Tuple):
 
 
 class PastCovariatesTorchModel(TorchForecastingModel, ABC):
+
+    uses_future_covariates = False
```

Comment:
Shouldn't this be True?
and is this used?

Reply:
from one of your earlier comments:
> Here's the list:
> * `PastCovariatesTorchModel`: only past
> * `FutureCovariatesTorchModel`, `DualCovariatesTorchModel` and `DualCovariatesForecastingModel`: only future
> * `MixedCovariatesTorchModel` and `SplitCovariatesTorchModel`: both

it is used [here](https://github.com/unit8co/darts/blob/d7f018904d7e19c510ca414d534322cd5b8fe4ac/darts/models/regression_ensemble_model.py#L119).

Reply:
Sorry, I read that too quick, my mistake.